### PR TITLE
#326

### LINKED ISSUE #326: Validation error shown twice on create account screen
The message `Câmpurile de parolă și confirmare parolă nu sunt identice.` is displayed twice on the screen.

Also, only the one below the input field is removed when the validation issue is fixed. The top one remains there even after updating the input field contents.

Screenshot from 2020-04-22 20-07-11

### PR DESCRIPTION
**Pull request by andorremus**
### Requirements for making a pull request

Thank you for contributing to our project! 

Please fill out the template below to help the project maintainers review it as fast as possible and include your contribution to the project.

### What does it fix?

Closes #326 

Adds email field validation messages (required and email)
Removes duplicate error messages on model.
Adds password required validation message.

### How has it been tested?

Please describe the tests that you ran to verify your changes.

### CODE REVIEW

File: backend/src/StamAcasa.IdentityServer/Areas/Identity/Pages/Account/Register.cshtml.cs [hunk 1]
```diff
@@ -54,17 +54,18 @@ public class RegisterModel : PageModel
 
         public class InputModel
         {
-            [Required]
-            [EmailAddress]
+            [Required(ErrorMessage = "Va rugam completati adresa de e-mail")]
+            [EmailAddress(ErrorMessage = "Adresa de e-mail nu este valida.")]
```
Comment (aniri) [inline backend/src/StamAcasa.IdentityServer/Areas/Identity/Pages/Account/Register.cshtml.cs]:
```suggestion
            [Required(ErrorMessage = "Te rugăm completează adresa de e-mail")]
            [EmailAddress(ErrorMessage = "Adresa de e-mail nu este validă.")]
```

File: backend/src/StamAcasa.IdentityServer/Areas/Identity/Pages/Account/Register.cshtml.cs [hunk 2]
```diff
@@ -54,17 +54,18 @@ public class RegisterModel : PageModel
...
             [Display(Name = "Adresa de e-mail")]
             public string Email { get; set; }
 
-            [Required]
-            [StringLength(100, ErrorMessage = "Câmpul {0} trebuie să aibă lungimea între minim {2} și maximum {1} caractere.", MinimumLength = 6)]
+            [Required(ErrorMessage = "Va rugam  sa completati parola")]
+            [StringLength(100, ErrorMessage = "Parola trebuie să aibă lungimea între minim {2} și maximum {1} caractere.", MinimumLength = 6)]
```
Comment (aniri) [inline backend/src/StamAcasa.IdentityServer/Areas/Identity/Pages/Account/Register.cshtml.cs]:
```suggestion
            [Required(ErrorMessage = "Te rugăm să completezi parola")]
            [StringLength(100, ErrorMessage = "Parola trebuie să aibă lungimea între minim {2} și maximum {1} caractere.", MinimumLength = 6)]
```

File: backend/src/StamAcasa.IdentityServer/Areas/Identity/Pages/Account/Register.cshtml.cs [hunk 3]
```diff
@@ -54,17 +54,18 @@ public class RegisterModel : PageModel
...
             [DataType(DataType.Password)]
             [Display(Name = "Setează parola")]
             public string Password { get; set; }
 
+            [Required(ErrorMessage = "Va rugam  sa completati confirmarea parolei")]
```
Comment (aniri) [inline backend/src/StamAcasa.IdentityServer/Areas/Identity/Pages/Account/Register.cshtml.cs]:
```suggestion
            [Required(ErrorMessage = "Te rugăm să completezi confirmarea parolei")]
```

**Review by aniri — COMMENTED** [review]
small wording suggestions and diacritics :)

**Comment by andorremus** [comment]
Hi @aniri, thanks for the review, I've merged your changes now.

Do I need to do a rebase again or is this ok now?

Update: NVM, done it pre-emptively.

**Review by idormenco — APPROVED** [review]

**Review by catileptic — APPROVED** [review]

**Pull request merged**